### ISSUE TITLE
Crash provoked by cheaters >3095

### ISSUE THREAD

**Issue by PichotM** [citizenfx/fivem#2709]
### What happened?

Cheaters can provoke a crash in 3095
I have no information I only know mostly freeroam & Brazilian pvp servers are impacted

`GTA5_b3095.exe!sub_14168B814`
`april-helium-arizona`

![image](https://github.com/user-attachments/assets/add629bc-8538-4172-b055-031f65b53db6)

[CfxCrashDump_2024_08_11_18_10_06 (1).zip](https://github.com/user-attachments/files/16588181/CfxCrashDump_2024_08_11_18_10_06.1.zip)

### Expected result

Not crash

### Reproduction steps

None

### Importancy

Crash

### Area(s)

FiveM

### Specific version(s)

3095

**Comment by slashkeyvalue** [comment]
Fixed by [052059c](https://github.com/citizenfx/fivem/commit/052059c2326a3d1222389037ffac57f32ba1890f)

**Comment by QBLes** [comment]
i dont think this is the fix, they are using any props for example the road cones are also used a lot by these people.. i know some servers that have had up to 60+ crashes in 24 hours based on these players doing this... all with code b3095

**Comment by slashkeyvalue** [comment]
> i dont think this is the fix, they are using any props for example the road cones are also used a lot by these people.. i know some servers that have had up to 60+ crashes in 24 hours based on these players doing this... all with code b3095

Care to provide any more info? i haven't had them anymore nor have received reports of.

**Comment by QBLes** [comment]
> > i dont think this is the fix, they are using any props for example the road cones are also used a lot by these people.. i know some servers that have had up to 60+ crashes in 24 hours based on these players doing this... all with code b3095
> 
> Care to provide any more info? i haven't had them anymore nor have received reports of.

Just watch a cheaters youtube like DoseOfCrown, from the info i have gatherd myself his making the player spawn the prop (one i seen coords was in the sky) and its causing them to crash, i have watched him run around crashing player after player.. at one time i seen him next to 8 people made them crash one by one.. the prop he used in that cased was the roadcone.

Here is the log i got from a server owner seconds after he did it to a player, this all i could get from it..

`{"type":"entityCreated","owner":"Mikeio/739","entity":985393,"popType":0,"model":-1036807324,"pos":["295.7","-582.2","390.7"],"script":null}`

**Comment by slashkeyvalue** [comment]
That doesn't help, if you have a crash hash and/or a crash dump, create another issue with that information. This one issue has had no more hits as fair as i'm aware, so it could just be another unrelated crash
